src/app/components/ideas-list/ideas-list.component.ts

```diff
@@ -11,10 +11,19 @@ import { IdeasService } from '../../core/services/backend/ideas.service';
 })
 export class IdeasListComponent {
   ideas: Idea[] = [];
+  topics: string[] = [
```

Comment:
Create a enum with the topic in a new file in models folder. 
`export enum Topics {
ALL = 'All',
TECHNOLOGICAL = 'Technological',
...
}`
Import and use this enum here and in the publish form.